packages/client/src/parse-callback.ts

```diff
@@ -10,7 +10,7 @@ export interface AuthenticationResult {
 export const parseRedirectCallback = (url: string): AuthenticationResult => {
   const [, queryString] = url.split('?');
```

Comment:
not sure if it is a better approach, but definitely shorter~   @wangsijie please check if this is a little bit hacky or not. 

as qs.parse('') will return empty object. 
```js
const [, queryString = ''] = url.split('?');
return qs.parse(queryString);
```

Reply:
@IceHe woulld you mind merge this change?

Reply:
That's all right. I will do it.